### PR TITLE
CLI for stopping write job

### PR DESCRIPTION
**Pull request by umithardal**
This PR introduces a standalone CLI for stopping a write job for a given `job_id`.

Basic usage:

- `python stop_file_writer.py -s <job_id> -b <broker> -t <topic>` to stop immediately
- `python stop_file_writer.py -sa <job_id> <time> -b <broker> -t <topic>` to stop after <time>

Here, `<broker>` defaults to `localhost` if not provided.

### CODE REVIEW

File: cli/stop_file_writer.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    args = fw_parser.parse_args()
+
+    return args
+
+
+def create_job_handler(args: argparse.Namespace, job_id: str) -> JobHandler:
+    host = args.broker
+    topic = args.topic
+    command_channel = WorkerCommandChannel(f"{host}/{topic}")
```
Comment (kmurica) [inline cli/stop_file_writer.py:67]: As we are moving away from creating jobs in the non-pool mode, I'd suggest using WorkerJobPool  here instead.
This also needs to be done for the CLI start file writer. Could you create a separate ticket for that case?

File: cli/stop_file_writer.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+def cli_parser() -> argparse.Namespace:
+    """
+    Parser for the command line interface.
+    """
+
+    fw_parser = argparse.ArgumentParser(
+        fromfile_prefix_chars="@", description="FileWriter Stopper"
+    )
+
+    fw_parser.add_argument(
```
Comment (kmurica) [inline cli/stop_file_writer.py:20]: I presume -s and --sa both expect a job id to stop the correct FW job? Would it be possible to add a cli command here to list available job id's that are subscribing to the command topic?

Comment (ebadkamil) [reply cli/stop_file_writer.py:20]: good idea. But may require looking for all start messages on the topic. or may be there is an easy way.

File: cli/stop_file_writer.py (hunk 1 above)
Comment (umithardal) [reply cli/stop_file_writer.py:67]: Should we wait until you remove it from the control library altogether? Or this PR should handle that?

Comment (kmurica) [reply cli/stop_file_writer.py:67]: I think doing it now makes sense. It will be easier for me to remove the non-pool jobs then. :)

**Comment by kmurica** [comment]
Except for my comment regarding using the job pool instead, it looks good. I confirmed that it works on my side.

File: cli/stop_file_writer.py (hunk 1 above)
Comment (umithardal) [reply cli/stop_file_writer.py:67]: Using job pool requires another Kafka topic which Jonas referred to as the Kafka topic that the available file-writers are listening to for write jobs. I am not sure if the command topic (that we currently use) can be use a job topic as well. 

Have you used this class before?

Comment (kmurica) [reply cli/stop_file_writer.py:67]: I think you just need to provide a topic name in the file writer config ini file (which is done by the end-user anyway) and create the topic before running the file writer.

Comment (umithardal) [reply cli/stop_file_writer.py:67]: So we do not need a special configuration other than defining it? Ok, I had the feeling may be we need something special.

Comment (ebadkamil) [reply cli/stop_file_writer.py:67]: According to `job_pool.py in examples`, one just have to provide an extra job-pool topic. Apart from that everything stays the same when we had direct communication via `WorkerCommandChannel`.

File: cli/stop_file_writer.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        "-b",
+        "--broker",
+        metavar="kafka_broker",
+        type=str,
+        default="localhost:9092",
+        help="Kafka broker port.",
+    )
+    fw_parser.add_argument(
+        "-t",
+        "--topic",
```
Comment (ebadkamil) [inline cli/stop_file_writer.py:45]: `--command-status-topic` may be. To stay consistent with the naming in FW. There it is called `command-status-uri`

File: cli/stop_file_writer.py (hunk 1 above)
Comment (umithardal) [reply cli/stop_file_writer.py:67]: Great. I was just sceptical if that specific topic should be something special or not (configured in a specific way). But not an issue it seems, will do that.

File: cli/stop_file_writer.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        metavar="consume_topic",
+        type=str,
+        required=True,
+        help="Name of the Kafka topic to be consumed.",
```
Comment (ebadkamil) [inline cli/stop_file_writer.py:49]: `"Name of the Kafka topic to listen to commands and send status to"`

File: cli/stop_file_writer.py (hunk 1 above)
Comment (ebadkamil) [reply cli/stop_file_writer.py:67]: yes it works like a charm.

File: cli/stop_file_writer.py (hunk 4 above)
Comment (ebadkamil) [reply cli/stop_file_writer.py:49]: FYI: I also made same changes in your `start_file_writer` script in https://github.com/ess-dmsc/file-writer-control/pull/27

Comment (umithardal) [reply cli/stop_file_writer.py:49]: Saw it now, thank you.

**Review by ebadkamil — APPROVED** [review]

File: cli/stop_file_writer.py (hunk 1 above)
Comment (umithardal) [reply cli/stop_file_writer.py:67]: I will merge this and make the pool changes in both CLIs simultaneously in a single PR. that is more proper and easy to test.

**Pull request merged**